CMakeLists.txt

```diff
@@ -98,10 +98,21 @@ function(RELATIVE_PROTOBUF_GENERATE_CPP SRCS HDRS ROOT_DIR)
     file(RELATIVE_PATH REL_DIR ${ROOT_DIR} ${FILE_DIR})
     set(OUTPUT_PROTO_DIR "${CMAKE_CURRENT_BINARY_DIR}/${REL_DIR}")
 
-    list(APPEND ${SRCS} "${OUTPUT_PROTO_DIR}/${FILE_WE}.pb.cc")
-    list(APPEND ${HDRS} "${OUTPUT_PROTO_DIR}/${FILE_WE}.pb.h")
+    if(ONNX_NAMESPACE STREQUAL "onnx")
+      set(OUTPUT_PB_HEADER "${OUTPUT_PROTO_DIR}/${FILE_WE}.pb.h")
+      set(OUTPUT_PB_SRC "${OUTPUT_PROTO_DIR}/${FILE_WE}.pb.cc")
+      set(GENERATED_PROTO "${OUTPUT_PROTO_DIR}/${FILE_WE}.proto")
+    else()
+      set(OUTPUT_PB_HEADER "${OUTPUT_PROTO_DIR}/${FILE_WE}_${ONNX_NAMESPACE}.pb.h")
+      set(OUTPUT_PB_SRC "${OUTPUT_PROTO_DIR}/${FILE_WE}_${ONNX_NAMESPACE}.pb.cc")
+      # We need this dummy header generated by gen_proto.py when ONNX_NAMESPACE
+      # is not onnx
+      list(APPEND ${HDRS} "${OUTPUT_PROTO_DIR}/${FILE_WE}.pb.h")
```

Comment:
this should be `_pb.h`?

Reply:
`_pb.h` is not in the same directory. It's included in L154 I think.

Reply:
lol does it mean we have a proxy to proxy now?

Reply:
Yes!

Reply:
Since we can't change `_pb.h`.